.github/workflows/dispatch-release-pypi.yml

```diff
@@ -3,8 +3,9 @@
 name: Release to PyPi
 
 on:
-  release:
-    types: [published]
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
When we create a release, we create a tag but this could also be more extensible outside of creating a release?

Otherwise we could limit this to the release again without issue

Reply:
Creating a PyPi release on GitHub release creation seems sensible to me. What benefits do we get from decorelating those actions?

Reply:
This will still happen for every release with a tag, but would allow for pushing a tag without a release.

In the past this has been useful for testing purposes without cluttering the releases page, however it may weaken clarity on pre-releases.

We could definitely revert this to work with release creation instead if that's preferred.

Reply:
Is there a need to push version tags without a release?

Reply:
No, not necessarily. Maybe keeping this tied this to releases is a better idea. 👍 

I'll change this trigger back to a release 👍

Reply:
Restored now 👍